pkg/hintrunner/zero/hintcode.go

```diff
@@ -7,4 +7,10 @@ const (
 	// the identifier resolution code.
 	// Depending on the context, ids.a may be a complex reference.
 	TestAssignCode string = "memory[ap] = ids.a"
+
+	// assert_le_felt() hints.
+	assertLeFelt          string = "import itertools\n\nfrom starkware.cairo.common.math_utils import assert_integer\nassert_integer(ids.a)\nassert_integer(ids.b)\na = ids.a % PRIME\nb = ids.b % PRIME\nassert a <= b, f'a = {a} is not less than or equal to b = {b}.'\n\n# Find an arc less than PRIME / 3, and another less than PRIME / 2.\nlengths_and_indices = [(a, 0), (b - a, 1), (PRIME - 1 - b, 2)]\nlengths_and_indices.sort()\nassert lengths_and_indices[0][0] <= PRIME // 3 and lengths_and_indices[1][0] <= PRIME // 2\nexcluded = lengths_and_indices[2][1]\n\nmemory[ids.range_check_ptr + 1], memory[ids.range_check_ptr + 0] = (\n    divmod(lengths_and_indices[0][0], ids.PRIME_OVER_3_HIGH))\nmemory[ids.range_check_ptr + 3], memory[ids.range_check_ptr + 2] = (\n    divmod(lengths_and_indices[1][0], ids.PRIME_OVER_2_HIGH))"
```

Comment:
I would like to follow some formatting with the name of these constants. The first two are public (which I think we should change to private since they are only supposed to be used in this package) and have the suffix "_Code" (which I would like to add to the others as I think is more self-explanatory)